src/hooks/useContributions.ts

```diff
@@ -38,7 +38,98 @@ const mockData: UserContributions = {
   ],
 };
 
-export function useContributionUsers(): UserContributions {
+export function useContributionUsers(timeInput: {
+  start_date: string;
+  end_date: string;
+}): UserContributions {
+  const integrations = useCurrentCircleIntegrations();
+  const responses = useQueries(
+    integrations.data
+      ? integrations.data
+          .filter(
+            integration =>
+              integration.type === 'dework' || integration.type === 'wonder'
+          )
+          .map(integration => ({
+            queryKey: `circle-integration-contributions-${integration.id}-${timeInput.start_date}-${timeInput.end_date}`,
+            queryFn: () => {
+              if (integration.type === 'dework') {
+                return fetch(
+                  `https://api.deworkxyz.com/integrations/coordinape/${
```

Comment:
this should probably be a const higher up in file

Reply:
and this whole block should be a func

Reply:
your wish is my command